gradle/libs.versions.toml

```diff
@@ -7,6 +7,7 @@ kotlinx-serialization = "1.9.0"
 vanniktech-publish = "0.34.0"
 dokka = "2.1.0-Beta"
 kotlinx-io = "0.8.0"
+semver = "0.8.0"
```

Comment:
v0.9.0 is the newest, but we need Gradle 9.0 for that, so therefore a future version of the kotlin gradle plugin.